manifest.py

```diff
@@ -42,7 +43,7 @@ def convert(self, value, param, ctx):
                 bands = [int(x) for x in value.split(',')]
                 assert len(bands) in (3, 4)
                 assert all(b > 0 for b in bands)
```

Comment:
@vincentsarago @normanb there's a problem with using `assert` here: assert statements are ignored if we run the validation with `python -O`. Since we can't control the optimization flags the user might use, it's better to test in some other way and then raise a manifest-specific exception if the test fails.